e2e/utils.go

```diff
@@ -425,6 +443,39 @@ func createRBDSecret(c kubernetes.Interface, f *framework.Framework) {
 	sc.Namespace = cephCSINamespace
 	_, err := c.CoreV1().Secrets(cephCSINamespace).Create(&sc)
 	Expect(err).Should(BeNil())
+
+	err = updateSecretForEncrypt(c)
+	Expect(err).Should(BeNil())
+}
+
+// updateSecretForEncrypt is an hack to update the secrets created by rook to
+// include the encyption key
+// TODO in cephcsi we need to create own users in ceph cluster and use it for E2E
```

Comment:
`TODO:`

Reply:
This todo cannot be fixed now

Reply:
@Madhu-1 what I mentioned was just to add `:` after TODO :)